Documentation/WritingDocsOfficial/HowYouCanHelp.rst

```diff
@@ -29,25 +29,84 @@ for a walkthrough.
 
 
 .. _how-you-can-help-fix-issues:
+.. _links-github-issues:
 
 Fix issues
 ==========
 
-See :ref:`links-github-issues`. As a new contributor, specifically look at issues
-with the label :ref:`good first issue <github-good-first-issue>`.
+Pick an issue for a topic you are familiar with and try to fix it. Some of the
+issues address a problem, some are enhancements where new text needs to be
+written.
+
+You can look at the open issues of a manual you are familiar with, pick one
+and fix the problem.
+
+For example:
+
+*  `Issues for this guide "Writing Documentation"
```

Comment:
I would rather provide links to TYPO3 Explained issues or TypoScript Reference issues -> they are more important and we have lots of them. What do you think?

Reply:
sure, it is more helpful, changed it